10.9-libcxx/stable/main/finkinfo/libs/perlmods/module-scandeps-pm-15.0.info

```diff
@@ -0,0 +1,34 @@
+Info2: <<
+Package: module-scandeps-pm
+Version: 1.31
+Revision: 702
```

Comment:
Please maintain +100 Revision spacing among distros. The module-scandeps-pm-14.0.info is Revision:701 for perl 5.30.3, so this file for perl 5.34.1 should be 801 not 702.